2200096/D3_소득불균형.py

```diff
@@ -0,0 +1,13 @@
+import sys
+sys.stdin = open("input.txt", "r")
+
+T = int(input())
+for t in range(1,T+1):
+    N = int(input())
+    numbers = list(map(int,input().split()))
+    AVG = int(sum(numbers)/N)

```

Comment:
좋은 풀이네요! 👍 
```suggestion
    AVG = sum(numbers)/N
```
정확한 평균 값은 `int` 없이 구하는 게 맞습니다! (이 문제에서는 정답과 관계 없습니다 😄)